docs/content/utilities/grid.md

```diff
@@ -9,6 +9,25 @@ bundle: layout
 
 The grid is 12 columns and percentage-based. The number of columns a container spans can be adjusted across breakpoints for responsive layouts. The grid system works with a variety of layout utilities to achieve different results.
 
+## Flexbox grids
```

Comment:
Maybe we should move the flexbox based grid before floats. Until we have a `d-grid` based grid. 😉